adm_program/installation/update.php

```diff
@@ -93,7 +94,7 @@
 $gProfileFields = new ProfileFields($gDb, $currOrgId);
```

Comment:
Why is this necessary?

This is the reason for the Notice if you have version 2.1 installed and perform an update:
> Notice: Undefined index: usf_name_intern in /Users/Markus/Sites/github/admidio/adm_program/system/classes/profilefields.php on line 457

If I comment this line update starts.

Reply:
@ximex what's about this?